### DISCUSSION TITLE
Can I configure output directory for *.verified.* and *.received.* files?

### DISCUSSION THREAD

**Discussion by peterrabo** [VerifyTests/Verify#797] [Q&A]
I am looking for a way to store the *.verified.txt and *.received.txt files in different directory than the default directory. Since I want to use the Verify tool on an environment with production-copied data, it’s not allowed to have the *.txt files stored locally.

Is there a way to configure the output directory to – for example – an Azure Storage Account? And, if so, is Verify able to compare the files within this storage account, without having it copied to the default directory again?

**Comment by SimonCropp** [comment]
no. this is not possible

**Comment by SimonCropp** [comment]
i think i misunderstood your question. this might work for you https://github.com/VerifyTests/Verify/blob/main/docs/naming.md#derivepathinfo
